Resources/Prototypes/Entities/Structures/gates.yml

```diff
@@ -51,6 +57,71 @@
           Nand: { state: nand }
           Xnor: { state: xnor }
 
+- type: entity
+  parent: LogicGateOr
+  id: LogicGateAnd
+  suffix: And
+  components:
+  - type: Sprite
+    layers:
+    - state: base
+    - state: and
+      map: [ "enum.LogicGateLayers.Gate" ]
```

Comment:
Does this work? I think it did not before and would spawn the gate visuals in the AND state always.

Reply:
yes, was tested in the game and everything seems fine